### DISCUSSION TITLE
Summing up a PO

### DISCUSSION THREAD

**Discussion by pacomms** [inventree/InvenTree#4233] [Q&A]
HI i have just started with inventree, i am trying to get my PO to list the item price and then ass up to a total price per line, then a total PO price lower down, i have managed to get it to display the item price by changing the notes entry to line.purchase_price , however i dont know howto the get the next box to basically sum decimal line.quantity and line.purchase_price together ? any ideas , there must be a simple way to do it like autosum just not simple for me hehe

Thanks
Kurt

**Comment by miggland** [comment]
Are you referring to creating a report?

Try `line.total_price`
